SitumWayfinding/Classes/Positioning/PositioningPresenter.swift

```diff
@@ -81,12 +81,11 @@ class PositioningPresenter: NSObject, SITLocationDelegate, SITDirectionsDelegate
         self.locationManagerUserLocation = nil
         self.lastOOBAlert = 0.0
         self.lastCalibrationAlert = 0.0
-        view?.stop()
+        view?.stopPositioningAndNavigation(error: nil)
```

Comment:
Realmente ese método non para o posicionamento. Chamase cando hai un erro ó posiconar(o que xa para o posicionamento de por si) , e dende aquí, onde se te fixas un par de liñas máis arriba no  self.locationManager.removeUpdates() parase o posicionamento.
Eu se cadra dividiría ese método en dous:
* Unha que limpa a UI/variables para que quede preparada para outras sesions de posicionamento
* A parte de parar a navefación en si